Python/Python.sublime-syntax

```diff
@@ -1050,3 +1255,42 @@ contexts:
   strings:
     - include: string_quoted_double
     - include: string_quoted_single
+
+  inline_for:
+    - match: \b(for)\b
+      scope: keyword.control.flow.for.generator.python
+      push:
+        - meta_scope: meta.expression.generator.python
+        - match: \b(in)\b
```

Comment:
Silly question, but does this `in` also cover comprehensions (list, dict, etc.) as well as generators?
```py
[function(item) for item in iterable]
```